scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -136,14 +136,22 @@ cray ims images update "$IMS_IMAGE_ID" \
         --link-etag "${MANIFEST_ETAG}" \
         --link-path "s3://boot-images/$IMS_IMAGE_ID/manifest.json" > /dev/null
 
-podman run --rm --name ncn-cpc \
+# shellcheck disable=SC2089
+PODMAN_RUN="podman run --rm --name ncn-cpc \
     --user root \
     -e PRODUCT=csm \
     -e PRODUCT_VERSION=$CSM_RELEASE \
-    -e YAML_CONTENT_STRING="{images: {\"$IMS_IMAGE_NAME\": {id: \"$IMS_IMAGE_ID\"}}}" \
+    -e YAML_CONTENT_STRING=\"{images: {\"$IMS_IMAGE_NAME\": {id: \"$IMS_IMAGE_ID\"}}}\" \
     -e KUBECONFIG=/.kube/admin.conf \
-    -e VALIDATE_SCHEMA="true" \
+    -e VALIDATE_SCHEMA=\"true\" \
     -v /etc/kubernetes:/.kube:ro \
-    registry.local/artifactory.algol60.net/csm-docker/stable/cray-product-catalog-update:$CPC_VERSION > /dev/null
+    registry.local/artifactory.algol60.net/csm-docker/stable/cray-product-catalog-update:$CPC_VERSION"
+
+# shellcheck disable=SC2090
+if test -f /etc/pit-release; then
+    ssh ncn-m002 $PODMAN_RUN >& /dev/null
```

Comment:
Could remove the hardcode by fetching the first-master hostname out of the `prep` directory.

```bash
FM=$(cat "${PITDATA}"/configs/data.json | jq -r '."Global"."meta-data"."first-master-hostname"')
echo ${FM}
```